examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>We could expose the parameter <code>k</code> in the model <code>call</code> so as to showcase that the threshold <code>k</code> is not fixed </p><pre class="ql-syntax" spellcheck="false">topk_model(batch[0], k=20)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>fixed.</p>